testhelpers/testhelpers.go

```diff
@@ -622,6 +583,10 @@ func RunContainer(ctx context.Context, dockerCli client.CommonAPIClient, id stri
 	if err := dockerCli.ContainerStart(ctx, id, dockertypes.ContainerStartOptions{}); err != nil {
 		return errors.Wrap(err, "container start")
 	}
+
+	// wait for logs to show
+	time.Sleep(time.Second)
```

Comment:
This would be dependant on the environment performance. 

Could we discuss why this was necessary to begin with? Maybe we can come up with a better solution.

Reply:
We added this because the container logs were always showing up empty on the WCOW runner (but not WCOW locally). We don't know the root cause, but it had to do with the container perhaps quitting too soon in some scenarios. At first, we thought it was because process isolation (`docker run --isolation=process` equivalent) was not enabled on the runner (it's enabled locally on our VMs), but that was proven wrong. There's something about the runner configuration that is causing logging capture to run more slowly than than it does locally.

We were originally considering putting this under a `dockerHostOS() == "windows"` guard, but didn't want to add yet another branch for that unless necessary. Happy to do that after all if you're concerned about performance.

Reply:
Maybe try wrapping in a context with a cancel? Like what we do [here](https://github.com/buildpacks/pack/blob/3a3bf282e703ee50c7f9e189015a3a9914b359a1/testhelpers/registry.go#L96) for the registry?

Reply:
@natalieparellano Could you elaborate on that suggestion? I'm a bit confused as we want to block for 1 second, not timeout after a certain amount of time. I could be missing something though :)

Reply:
I would hate for this to be the reason this doesn't make it to the next release. For now, can we guard it as you suggested and create an issue so that we can revisit it afterwards.